### PR TITLE
Add initial README (taken from nucypher/nucypher docs)

### LINKED ISSUE #8: Author Readme.md

### PR DESCRIPTION
**Pull request by derekpierre**
Based over #9.
Fixes #8

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,730 @@
+Porter
+======
+
+Overview
+--------
+NuCypher Porter can be described as the *“Infura for NuCypher”*. Porter is a web-based service that performs
+nucypher-based protocol operations on behalf of applications.
```
Comment (KPrasch) [inline README.rst:7]: Just a thought to sprinkle in somewhere:

*This enables applications to behave like a "light client" by delegating network intensive operations like peering and node discovery to porter*

**Review by KPrasch — APPROVED** [review]
a few minor suggestions

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,730 @@
...
+
+Its goal is to simplify and abstract the complexities surrounding the nucypher protocol to negate the need for
+applications to interact with it via a python client. Porter introduces the nucypher protocol to cross-platform
```
Comment (KPrasch) [inline README.rst:10]:
```suggestion
applications to interact with the network directly. Porter introduces the nucypher protocol to cross-platform
```

Comment (KPrasch) [reply README.rst:10]: > Porter introduces the nucypher protocol to cross-platform functionality including web and mobile applications.

With the evolution of nucypher-ts this is not the motivation anymore IMO.

File: README.rst (hunk 1 above)
Comment (derekpierre) [reply README.rst:7]: 👍

File: README.rst (hunk 2 above)
Comment (derekpierre) [reply README.rst]: 👍

**Review by KPrasch — COMMENTED** [review]
LGTM

**Pull request merged**